spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-grpc/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+# This file uses the version 2 docker-compose file format, described here:
+# https://docs.docker.com/compose/compose-file/#version-2
+#
+# This runs the zipkin and zipkin-mysql containers, using docker-compose's
+# default networking to wire the containers together.
+#
+# Note that this file is meant for learning Zipkin, not production deployments.
+
+version: '2'
+
+services:
+  storage:
```

Comment:
mysql is unnecessary for this.. just use default in-mem.. probably docker-compose itself is overkill

See also https://github.com/openzipkin/sleuth-webmvc-example

Reply:
I will clean that up as well, thanks.